### ISSUE TITLE
Tensorflow chatbot rnn model test issue

### ISSUE THREAD

**Issue by rplawate** [tensorflow/tensorflow#23701]
I am getting the following error while testing seq2seq rnn model for chatbot

`Key embedding_attention_seq2seq/embedding_attention_decoder/attention_decoder/Attention_0/bias not found in checkpoint
 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_HALF, DT_INT32, DT_HALF, DT_HALF, DT_HALF, ..., DT_HALF, DT_HALF, DT_HALF, DT_HALF, DT_HALF], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
 [[Node: save/RestoreV2/_1 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_6_save/RestoreV2", tensor_type=DT_HALF, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]`

**Comment by wt-huang** [comment]
Please provide more details by filling out the issue template. 
Did you use a pre-trained model?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @wt-huang: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
